torch/csrc/Module.cpp

```diff
@@ -331,6 +331,16 @@ PyObject* THPModule_setDefaultDtype(PyObject* _unused, PyObject* dtype) {
   END_HANDLE_TH_ERRORS
 }
 
+PyObject* THPModule__tensor_use_count(PyObject* _unused, PyObject* cdata) {
+  HANDLE_TH_ERRORS
+  auto tensor_impl_ptr = THPUtils_unpackLong(cdata);
+  // NOLINTNEXTLINE(performance-no-int-to-ptr)
+  auto tensor_impl = (c10::TensorImpl*)tensor_impl_ptr;
```

Comment:
Huh, this is not a raw pointer to a TensorImpl! It's an intrusive_ptr to a TensorImpl! There are more fields inlined!